pandas/core/computation/parsing.py

```diff
@@ -0,0 +1,159 @@
+""":func:`~pandas.eval` source string parsing functions
+"""
+
+from io import StringIO
+from keyword import iskeyword
+import token
+import tokenize
+from typing import Iterator, Tuple, Union
+
+# A token value Python's tokenizer probably will never use.
+BACKTICK_QUOTED_STRING = 100
+
+Tok = Tuple[int, str]
+
+
+def create_valid_python_identifier(name: str) -> str:
+    """Create valid Python identifiers from any string.
```

Comment:
newline after triple quotes

Reply:
Well, this is not what PEP 257 suggested. I don't think it is very strict, but I also see both styles in the pandas code base, so I picked the one suggested in the PEP. 

https://www.python.org/dev/peps/pep-0257/

Reply:
Well, this is not what PEP 257 suggested. I don't think it is very strict, but I also see both styles in the pandas code base, so I picked the one suggested in the PEP. 

https://www.python.org/dev/peps/pep-0257/

Reply:
Okay, changed it to that now.